<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Produit</class>
 <widget class="QWidget" name="Produit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/ADMIN/Stock.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>1260</width>
     <height>630</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="Ajout">
    <attribute name="title">
     <string>Ajouter Produit</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantite</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix Unitaire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="quantite">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>390</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Nom_Produit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Reference">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom Produit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Prix_u">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>490</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonAjouter.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="AjouterProduit">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>490</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Afficherr">
    <attribute name="title">
     <string>Afficher Stock</string>
    </attribute>
    <widget class="QTableWidget" name="Afficher">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>551</width>
       <height>511</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <column>
      <property name="text">
       <string>Reference</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prix_Unitaire</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom Produit</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantite</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>601</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="Lref_2">
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modif_ref"/>
     <widget class="QLabel" name="lNom">
      <property name="text">
       <string>Nom Produit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modif_nom"/>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>170</y>
       <width>278</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Quantite</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modif_qte"/>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>210</y>
       <width>310</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="m">
      <property name="text">
       <string>Prix Unitaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modif_prix"/>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>330</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonSupprimer.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>410</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonModifier.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>340</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>420</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Commande</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherC">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>781</width>
       <height>471</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID_Personne</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reference_Produit</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date_Commande</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Abonne">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Abonnement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Cours">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Promotion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>910</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
